sdk/schemaregistry/schema-registry-avro/test/public/utils/mockedRegistryClient.ts

```diff
@@ -137,3 +145,41 @@ export function createTestRegistry(
       })
     : createMockedTestRegistry();
 }
+export function createPipelineWithCredential(): Pipeline {
+  const DEFAULT_SCOPE = "https://eventhubs.azure.net/.default";
+  const pipeline = createEmptyPipeline();
+  const credential = createTestCredential();
+  const authPolicy = bearerTokenAuthenticationPolicy({ credential, scopes: DEFAULT_SCOPE });
+  pipeline.addPolicy(authPolicy);
+  return pipeline;
+}
+
+export async function removeSchemas(
+  schemaNamesList: string[],
+  pipeline: Pipeline,
+  client: HttpClient
+): Promise<void> {
+  if (!isLiveMode()) {
+    return;
+  }
+
+  function formatRequest(schemaName: string, apiVersion: string = "2022-10"): PipelineRequest {
+    const endpoint = assertEnvironmentVariable("SCHEMAREGISTRY_AVRO_FULLY_QUALIFIED_NAMESPACE");
+    const url = `${endpoint}/$schemagroups/${testGroup}/schemas/${schemaName}/?api-version=${apiVersion}`;
+    return createPipelineRequest({
+      url,
+      method: "DELETE",
+      timeout: 0,
+      withCredentials: true,
+      headers: createHttpHeaders({}),
+      allowInsecureConnection: new URL(url).protocol !== "https:",
```

Comment:
why is this needed?

Reply:
It's not. I forgot to remove it when bringing the function over.